### PR TITLE
Update about-admin-roles.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated security guidelines as Privileged authentication admin can also reset GA password

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10088

File: microsoft-365/admin/add-users/about-admin-roles.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Because admins have access to sensitive data and files, we recommend that you fo
 
 | Recommendation                  | Why is this important?                 |
 | :------------------- | :------------------- |
-| Have 2 to 4 global admins  | Because only another global admin can reset a global admin's password, we recommend that you have at least 2 global admins in your organization in case of account lockout. But the global admin has almost unlimited access to your org's settings and most of the data, so we also recommend that you don't have more than 4 global admins because that's a security threat. |
+| Have 2 to 4 global admins  | Global Admins have almost unlimited access to your organization's settings and most of its data. We recommend you limit the number of Global Admins as much as possible. A Global Admin may inadvertently lock their account and require a password reset. Either another Global Admin or a Privileged Authentication Admin can reset a Global Admin's password. Therefore, we recommend you have at least either one more Global Admin or a Privileged Authentication Admin in the event a Global Admin locks their account. |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/about-admin-roles.md]:
```suggestion
| Have 2 to 4 Global Admins  | Global Admins have almost unlimited access to your organization's settings and most of its data. We recommend you limit the number of Global Admins as much as possible. A Global Admin may inadvertently lock their account and require a password reset. Either another Global Admin or a Privileged Authentication Admin can reset a Global Admin's password. Therefore, we recommend you have at least either one more Global Admin or a Privileged Authentication Admin in the event a Global Admin locks their account. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**